docs/kbs_attestation_protocol.md

```diff
@@ -288,7 +289,7 @@ The authentication service is provided by the KBS through two endpoints:
 2. `/kbs/v0/attest` only accepts `POST` requests whose body is a [KBS Attestation](#attestation)
    JSON payload and the header contains a `Cookie` set to the value received in
    step 1.i. This is how the attester replies to attestation challenge received
-   in step 1.ii. If the attestation was successful, the KBS replies to that request with an 
+   in step 1.ii. If the attestation was successful, the KBS replies to that request with an
```

Comment:
You have a few cosmetic/formatting changes that are unrelated to this PR. Could you please keep those aside so that we dont mix two different set of change in the same PR?

Reply:
Sorry, my bad.